lib/sisimai/bite/email/sendgrid.rb

```diff
@@ -0,0 +1,223 @@
+module Sisimai::Bite::Email
+  module SendGrid
+    # Sisimai::Bite::Email::SendGrid parses a bounce email which created by
+    # SendGrid. Methods in the module are called from only Sisimai::Message.
+    class << self
+    # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/SendGrid.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :'from'        => %r/\AMAILER-DAEMON\z/,
+        :'return-path' => %r/\A[<]apps[@]sendgrid[.]net[>]\z/,
+        :'subject'     => %r/\AUndelivered Mail Returned to Sender\z/,
```

Comment:
__[rubocop]__
* Style/SymbolLiteral: Do not use strings for word-like symbol literals.
* Layout/SpaceAroundOperators: Operator `=>` should be surrounded by a single space. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

<!-- Comment by [SideCI](https://sideci.com) -->